docs/sources/setup-grafana/installation/kubernetes/index.md

```diff
@@ -9,114 +9,624 @@ weight: 500
 
 # Deploy Grafana on Kubernetes
 
-This page explains how to install and run Grafana on Kubernetes (K8S). It uses Kubernetes manifests for the setup. If you prefer Helm, refer to the [Grafana Helm community charts](https://github.com/grafana/helm-charts).
+This page explains how to install and run Grafana on Kubernetes (K8s). It uses Kubernetes manifests for the setup. If you prefer Helm, refer to the [Grafana Helm community charts](https://github.com/grafana/helm-charts).
+
+## Before you begin
+
+To follow this guide, you need:
+
+- The latest version of [Kubernetes](https://kubernetes.io/) running either locally or remotely (on a public or private cloud).
+
+- To use in a local environment use any Kubernetes such as [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), [Docker Desktop](https://docs.docker.com/desktop/kubernetes/), and so on.
+
+- To use managed cloud services such as [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine), [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/), or [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/products/kubernetes-service/) if you are considering using Kubernetes in a production environment.
+
+## System Requirements
+
+This section provides minimum hardware and software requirements.
+
+### Minimum Hardware Requirements
+
+- Disk Space = 1 GB
+- Memory = 750 MiB (approx 750 MB)
+- CPU = 250m (approx 2.5 cores)
+
+### Supported databases
+
+For a list of supported databases, refer to [supported databases](https://grafana.com/docs/grafana/latest/setup-grafana/installation/#supported-databases).
+
+### Supported web browsers
+
+For a list of support web browsers, refer to [supported web browsers](https://grafana.com/docs/grafana/latest/setup-grafana/installation/#supported-web-browsers).
+
+> **Note:** Make sure to enable port `3000` in your network environment as this is the Grafana default port.
 
 ## Deploy Grafana OSS on Kubernetes
 
-This section explains how to install Grafana using Kubernetes.
-If you are interested in the Grafana Enterprise version of this information, see [Deploy Grafana Enterprise on Kubernetes](#deploy-grafana-enterprise-on-kubernetes).
+This section explains how to install Grafana OSS using Kubernetes. If you want to install Grafana Enterprise on Kubernetes,  refer to [Deploy Grafana Enterprise on Kubernetes](https://grafana.com/docs/grafana/latest/setup-grafana/installation/kubernetes/#deploy-grafana-enterprise-on-kubernetes).
+
+When you deploy an application in Kubernetes, it will use its default namespace which may have other applications running and can cause conflicts and other issues.
```

Comment:
```suggestion
If you deploy an application in Kubernetes, it will use the default namespace which may already have other applications running. This can result in conflicts and other issues.
```